File: Ted.Server.Web/wwwroot/Auxiliary/ExceptionCodes.js
Ext.define('Ted.Enum.ExceptionCodes',
{
Generic : 0,

Authentication : 100,
NotSuperUser:101,
UserExist : 102,
PageNotFound : 103,
}
);
var ExceptionCodes = new Ext.create('Ted.Enum.ExceptionCodes');

File: Exceptions/ExceptionCodes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Ted.Server.Exceptions
{
    public enum ExceptionCodes : int
    {
        Generic = 0,

        Authentication = 100,
        NotSuperUser=101,
        UserExist = 102,
		PageNotFound = 103,
        TableNotFound = 104,
        ColumnExist = 105
    }
}
